src/App.js

```diff
@@ -8,6 +8,7 @@ import Experiment from './pages/Experiment';
 import Final from './pages/Final';
 import Review from './pages/Review';
 import Export from './pages/Export';
+import Unauthorized from './pages/Unauthorized';
```

Comment:
it might be nitpick but `Unauthorized` usually means a little bit different thing (get unauthorized = logged out).
Could you please rename it to `Forbidden` please? Also, it would be more consistent with backend where we use `401 Unauthorized` to logout user and `403 Forbidden` to disallow to see protected content.

Reply:
Yes, it makes sense, I've changed it in 5b2d777